### ISSUE TITLE
could we add 'title' property so I can set the spoon Report html name?

### ISSUE THREAD

**Issue by chthompson** [stanfy/spoon-gradle-plugin#77]
Hey guys,

Is it possible to wire up the title arg so we can name tweak the spoon report html title name?

This would be great.

**Comment by roman-mazur** [comment]
Yes, the change should be trivial: exposing title to the spoon extension and using the spoon runner builder method:
https://github.com/square/spoon/blob/parent-1.2.0/spoon-runner/src/main/java/com/squareup/spoon/SpoonRunner.java#L252-L256
